src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -25,19 +23,30 @@
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.person.NameContainsKeywordsPredicate;
 import seedu.address.model.person.Person;
+import seedu.address.model.person.Student;
+import seedu.address.model.person.Tutor;
 import seedu.address.testutil.EditPersonDescriptorBuilder;
-import seedu.address.testutil.PersonBuilder;
 import seedu.address.testutil.PersonUtil;
+import seedu.address.testutil.StudentBuilder;
+import seedu.address.testutil.StudentUtil;
+import seedu.address.testutil.TutorBuilder;
+import seedu.address.testutil.TutorUtil;
 
 public class AddressBookParserTest {
 
     private final AddressBookParser parser = new AddressBookParser();
 
     @Test
     public void parseCommand_add() throws Exception {
-        Person person = new PersonBuilder().build();
-        AddCommand command = (AddCommand) parser.parseCommand(PersonUtil.getAddCommand(person));
-        assertEquals(new AddCommand(person), command);
+        Tutor tutor = new TutorBuilder().build();
+        Student student = new StudentBuilder().build();
+
+        AddCommand commandTutor = (AddCommand) parser.parseCommand(TutorUtil.getAddCommand(tutor, PersonType.TUTOR));
```

Comment:
I think `TutorUtil.getAddCommand()` accepts 1 argument, so this will be calling `PersonUtil.getAddCommand()` instead